account-kit/signer/src/base.ts

```diff
@@ -312,6 +316,42 @@ export abstract class BaseAlchemySigner<TClient extends BaseSignerClient>
     }
   );
 
+  authenticateStep: (params: AuthParams) => Promise<AuthStepResult> =
```

Comment:
nit because this also starts and progresses the flow should the name be `handleAuth`, `authenticateOrAdvance`, or `progressAuth`? 

Should we break up starting and progressing authentication into two calls? Such as `login` and `verify`?